src/GraphQL/Types/Composite/ListGraphType.cs

```diff
@@ -27,15 +27,6 @@ protected ListGraphType(Type type)
 
         public IGraphType ResolvedType { get; set; }
 
-        public override string CollectTypes(TypeCollectionContext context)
-        {
-            var innerType = context.ResolveType(Type);
-            ResolvedType = innerType;
-            var name = innerType.CollectTypes(context);
-            context.AddType(name, innerType, context);
-            return "[{0}]".ToFormat(name);
```

Comment:
This string modification is not actually required as well as one for `NonNullGraphType`.